platforms/mqtt/mqtt_adaptor.go

```diff
@@ -33,7 +33,7 @@ type Adaptor struct {
 	clientKey     string
 	autoReconnect bool
 	cleanSession  bool
-	client        paho.Client
+	Client        paho.Client
```

Comment:
maybe it is better to create a getter instead?

Reply:
That is probably a good idea. We likely do not want to replace the entire client, just be able to call into it.